js/assignment01.js

```diff
@@ -0,0 +1,7 @@
+$(document).ready (function (){
+
+$('#first-name').text("Scott");
+$('#last-name').text("Fincham");
+
+$('.mcc').css('color', 'blue') .css('font-style', 'italic');
+});

```

Comment:
This file is not referenced by your html. So I would delete it to avoid confusion.